packages/sdk-metrics/src/state/DeltaMetricProcessor.ts

```diff
@@ -31,15 +31,30 @@ export class DeltaMetricProcessor<T extends Maybe<Accumulation>> {
   // TODO: find a reasonable mean to clean the memo;
   // https://github.com/open-telemetry/opentelemetry-specification/pull/2208
   private _cumulativeMemoStorage = new AttributeHashMap<T>();
-
-  constructor(private _aggregator: Aggregator<T>) {}
+  private _cardinalityLimit: number;
+  constructor(
+    private _aggregator: Aggregator<T>,
+    aggregationCardinalityLimit?: number
+  ) {
+    this._cardinalityLimit = aggregationCardinalityLimit ?? 2000;
```

Comment:
I think this should be 
```suggestion
    this._cardinalityLimit = (aggregationCardinalityLimit ?? 2000) - 1;
```

as the last one would be used up for the overflow accumulation, right?